doc/sphinx/addendum/universe-polymorphism.rst

```diff
@@ -279,6 +279,8 @@ Cumulativity Weak Constraints
 
 .. flag:: Cumulativity Weak Constraints
 
+   .. warning:: The status of this flag is experimental
```

Comment:
Here experimental means it's an unstable feature
Admittedly I think we haven't made much effort in stabilizing the heuristics of universe constraints in general so specifically warning for this one heuristic may be questionable